core/store/migrate/migrations/0200_default_values_for_last_finalized_block.sql.sql

```diff
@@ -0,0 +1,33 @@
+-- +goose Up
+
+WITH variables AS (
```

Comment:
Similar performance to the previous Go-based solution

2 chains, 100k records
```
BenchmarkBackfillingRecordsWithMigration200
BenchmarkBackfillingRecordsWithMigration200-12    	       6	1881018785 ns/op
BenchmarkBackfillingRecordsWithMigration200-12    	       7	1820046643 ns/op
BenchmarkBackfillingRecordsWithMigration200-12    	       7	1955067827 ns/op
BenchmarkBackfillingRecordsWithMigration200-12    	       7	1909758488 ns/op
BenchmarkBackfillingRecordsWithMigration200-12    	       7	1899759274 ns/op
```

Around 0,9s for a single chain